db/data/20240117114102_dedupe_icddrb_medical_histories.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class DedupeIcddrbMedicalHistories < ActiveRecord::Migration[6.1]
+  FACILITY_ID = "f472c5db-188f-4563-9bc7-9f86a6ed6403"
```

Comment:
Do we need to do this for other facilities/orgs also?

Reply:
nope, nobody else has sent multiple conditions per patient, I checked this